README.md

```diff
@@ -1,6 +1,13 @@
 # wildfly
 
-[![Build Status](https://travis-ci.org/biemond/biemond-wildfly.svg?branch=master)](https://travis-ci.org/biemond/biemond-wildfly) ![Puppet Forge Endorsement](https://img.shields.io/puppetforge/e/biemond/wildfly.svg) ![Puppet Forge Version](https://img.shields.io/puppetforge/v/biemond/wildfly.svg) ![Puppet Forge Score](https://img.shields.io/puppetforge/f/biemond/wildfly.svg)  ![Puppet Forge Downloads](http://img.shields.io/puppetforge/dt/biemond/wildfly.svg) [![Code Climate](https://codeclimate.com/github/biemond/biemond-wildfly/badges/gpa.svg)](https://codeclimate.com/github/biemond/biemond-wildfly) [![Coverage Status](https://coveralls.io/repos/biemond/biemond-wildfly/badge.svg?branch=master&service=github)](https://coveralls.io/github/biemond/biemond-wildfly?branch=master)
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-catalog-diff-viewer.svg)](https://github.com/voxpupuli/puppet-catalog-diff-viewer/blob/master/LICENSE)
+[![CI](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml/badge.svg)](https://github.com/voxpupuli/puppet-wildfly/actions/workflows/ci.yml)
+[![Donated by biemond](https://img.shields.io/badge/donated%20by-biemond-fb7047.svg)](#transfer-notice)
+
+![Puppet Forge Endorsement](https://img.shields.io/puppetforge/e/voxpupuli/wildfly)
+![Puppet Forge Version](https://img.shields.io/puppetforge/v/voxpupuli/wildfly)
+![Puppet Forge Score](https://img.shields.io/puppetforge/f/voxpupuli/wildfly)
+![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/voxpupuli/wildfly)
```

Comment:
you can also add:
```
[![puppetmodule.info docs](https://www.puppetmodule.info/images/badge.png)](https://www.puppetmodule.info/m/puppet-wildfly)
```

Reply:
will do

Reply:
ah thats this one, i removed it, because it looked broken